web/src/pages/Settings.tsx

```diff
@@ -43,21 +43,23 @@ import { useInitialCameraState } from "@/api/ws";
 import { isInIframe } from "@/utils/isIFrame";
 import { isPWA } from "@/utils/isPWA";
 import { useIsAdmin } from "@/hooks/use-is-admin";
+import { useTranslation } from "react-i18next";
 
 const allSettingsViews = [
-  "UI settings",
-  "explore settings",
-  "camera settings",
-  "masks / zones",
-  "motion tuner",
+  "uiSettings",
+  "exploreSettings",
+  "cameraSettings",
+  "masksAndZones",
```

Comment:
If you're renaming these, you should also update line 467 of ObjectLifecycle.tsx to 

`/settings?page=masksAndZones&camera=${event.camera}&object_mask=${eventSequence?.[current].data.box}`